[PLAY-21] adding-opacity-in-playbook-review

cvargaspower opened the pull request:
#### Screens

![](https://p-a6fbdk.t4.n0.cdn.getcloudapp.com/items/04u5615k/750e701c-ae88-4e2b-8136-3a9b16f7ecad.png?source=viewer&v=c1782cc0c6fc40b18c70e30ec795257a)

#### Breaking Changes
Need to change the opacity order correctly. Right now its 1 being transparent when 10 should be transparent. 

#### Process Changes
- Used the data_1 with primary bg-color to create the display/look I wanted. 
- Added an example Import in SCSS (couldn't figure out how to make it dark bg like the example) 
- Added a table with all the variables and code

#### Runway Ticket URL

[ [PLAY 21] ](https://nitro.powerhrg.com/runway/backlog_items/PLAY-21)

#### How to test this

[INSERT TESTING DETAILS]

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

jasoncypret commented:
@cvargaspower I'm going to give you some updates on the ticket...but here's some code you may need. https://share.getcloudapp.com/p9uNLD4Z

jasoncypret reviewed (changes requested):
Great job. Just some additional feedback for you. I worked up a quick screenshot with direct annotations below. Please address these:

![](https://p199.p4.n0.cdn.getcloudapp.com/items/eDugrwAe/56281cd2-2858-4a6f-b717-dad4c2aaadb1.jpg?v=9a1caba42b3f05100386856fe69eb4a8)

You this mock and the one attached to the story as a guide:

![](https://p199.p4.n0.cdn.getcloudapp.com/items/9ZuRKBRe/2343363c-ec1a-4507-9c07-5ff97302f4a7.jpg?v=979c8e6c627708071e33c5f2b4cfa57c)
Here’s this code: [example.erb](https://share.getcloudapp.com/9ZuRKBQe)

jasoncypret commented:
@cvargaspower 

1. Can you add the correct label? See the guidelines for that.
2. Also, can you start your story with the runway number in brackets. ex [PLAY-21]
3. Can you make your title more clear? These titles are used for our changelog. What are you adding/improving?

cvargaspower commented on playbook-website/app/javascript/site_styles/_visual_guidelines.scss:
```diff
@@ -42,4 +44,35 @@
     .ten { margin-left: 450px; }
   }
+
+  .opacity_1{
+    opacity: $opacity_1;
+  }
+  .opacity_2{
+    opacity: $opacity_2;
+  }
+  .opacity_3{
+    opacity: $opacity_3;
+  }
+  .opacity_4{
+    opacity: $opacity_4;
+  }
+  .opacity_5{
+    opacity: $opacity_5;
+  }
+  .opacity_6{
+    opacity: $opacity_6;
+  }
+  .opacity_7{
+    opacity: $opacity_7;
+  }
+  .opacity_8{
+    opacity: $opacity_8;
+  }
+  .opacity_9{
+    opacity: $opacity_9;
+  }
+  .opacity_10{
+    opacity: $opacity_10;
+  }
```
For my feedback, can you explain what you mean the 10, 20, etc? Did you want me to change the opacity to those numbers starting from 10?

jasoncypret replied:
For `$opacity_1` what is the value of that? for `$opacity_2` what is the value of that? Display those values there

benlangfeld commented on playbook-website/app/views/pages/visual_guidelines.html.erb:
```diff
@@ -14,9 +14,12 @@
       <%= render "pages/visual_guidelines_partials/positioning.html.slim" %>
       <%= render "pages/visual_guidelines_partials/spacing.html.slim" %>
   <% end %>
+
+ 
```
```suggestion
```

benlangfeld commented on playbook-website/app/javascript/site_styles/_visual_guidelines.scss:
```diff
@@ -42,4 +44,6 @@
     .nine { margin-left: 400px; }
     .ten { margin-left: 450px; }
   }
+
+ 
```
```suggestion
```

benlangfeld commented on playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:
```diff
@@ -0,0 +1,35 @@
+    opacity: $opacity_10;
+  }
+
+}
```
```suggestion
}

```

benlangfeld commented on playbook-website/app/javascript/site_styles/_visual_guidelines.scss:
```diff
@@ -42,4 +44,6 @@
   }
+
+ 
 }
```
```suggestion
}

```

cvargaspower commented:
#### Screens

![](https://p-a6fbdk.t4.n0.cdn.getcloudapp.com/items/eDugEQXB/755e2c53-0c1b-4cdf-bde6-b46fe4e20559.png?source=viewer&v=3f1fb97188eb4c9257d0b39e77c60474) 

#### Breaking Changes
- Need to change the opacity order correctly. Right now its 1 being transparent when 10 should be transparent. 

#### Fixed Changes
- added a separate _opacity_guideliness.scss within the style-css folder, BUT it wouldn't connect after import so I reverted back to what the code was before the change. The file is still located within the PR, but just wanted to make that note. 
- Added a description
- Added a description on how to use Tokens
- Added a title so users know where they are: i.e opacity tokens
- Added the correct highlight coded sample.

#### Runway Ticket URL

[ [PLAY 21] ](https://nitro.powerhrg.com/runway/backlog_items/PLAY-21)

#### How to test this

[INSERT TESTING DETAILS]

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

benlangfeld commented on playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:
```diff
@@ -0,0 +1,38 @@
+
+
+
```
```suggestion
```

benlangfeld commented on playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:
```diff
@@ -0,0 +1,38 @@
+  .opacity_10{
+    opacity: $opacity_10;
+  }
+  
+
```
```suggestion
```

benlangfeld commented on playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:
```diff
@@ -0,0 +1,38 @@
+
+
+.selector { 
+    .opacity_1{
```
```suggestion
  .opacity_1{
```

benlangfeld commented on playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:
```diff
@@ -0,0 +1,38 @@
+  
+
+}
+
```
```suggestion
```

cvargaspower commented:
@benlangfeld I pushed another/committed with the space changes, hopefully all the coding spaces are good to go. Sorry about that... it must have happened when I was copy/pasting/moving things around.

kre8sions commented:
@cvargaspower  lets try to make the example look like this with the dark area and the card connected

<img width="541" alt="Screen Shot 2021-11-18 at 12 54 46 PM" src="https://user-images.githubusercontent.com/9784112/142470175-44f3f4e3-14cb-464b-ae46-08010dbf5369.png">

cvargaspower commented:
#### Screens

![](https://p-a6fbdk.t4.n0.cdn.getcloudapp.com/items/4guPRw78/e0894491-f38c-486e-a2aa-52b9a797248b.png?source=viewer&v=849fba5eba2f8d6b0e71f26e929460c7) 

#### Breaking Changes
- Need to change the opacity order correctly. Right now its 1 being transparent when 10 should be transparent. 

#### Fixed Changes
- Whitespaces have been fixed and space with dark card and light card also fixed!
- added a separate _opacity_guideliness.scss within the style-css folder, BUT it wouldn't connect after import so I reverted back to what the code was before the change. The file is still located within the PR, but just wanted to make that note. 
- Added a description
- Added a description on how to use Tokens
- Added a title so users know where they are: i.e opacity tokens
- Added the correct highlight coded sample.

#### Runway Ticket URL

[ [PLAY 21] ](https://nitro.powerhrg.com/runway/backlog_items/PLAY-21)

#### How to test this

[INSERT TESTING DETAILS]

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

jasoncypret commented on playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:
```diff
@@ -0,0 +1,48 @@
+  }
+}
+
+.selector { 
```
Lets improve this class name? What is this? is this a container of content? perhaps it is containing the opacity guidelines. Why not name it `.opacity-guideline-container {`

jasoncypret commented on playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:
```diff
@@ -0,0 +1,48 @@
+ul {
```
Give this a class name. This will affect EVERY list on the site. Major style bleed.

thestephenmarshall replied:
Agreed. @cvargaspower specificity is of great importance any time you are working with CSS unless you are attempting to style everything in the browser view. This selector would potentially have unintended consequences as you can reference from the CSS courses.

jasoncypret commented on playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:
```diff
@@ -0,0 +1,48 @@
+ul {
+   
+  &.opacity-play {
+  list-style: none;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin: none;
+  background: none;
+  border-radius: none;
+  border: 0px;
+  padding: none;
+  flex-wrap: wrap;
```
Indendation
```suggestion
    list-style: none;
    display: flex;
    justify-content: flex-start;
    align-items: center;
    margin: none;
    background: none;
    border-radius: none;
    border: 0px;
    padding: none;
    flex-wrap: wrap;
```

thestephenmarshall replied:
@cvargaspower did you install the VSCode extension `Beautify`?

<img width="410" alt="Screen Shot 2021-11-19 at 8 46 57 AM" src="https://user-images.githubusercontent.com/2293844/142641702-5cb72b2d-8351-4ee6-b56f-a3ab022d2e3b.png">

If not, you will need to do that now. On your working files, you should right click and choose "Format Document" and these things (as mentioned above) will auto-fix.

jasoncypret commented on playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:
```diff
@@ -0,0 +1,48 @@
+}
+
+.selector { 
+  .opacity_1{
```
```suggestion
  .opacity_1 {
```

jasoncypret commented on playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:
```diff
@@ -0,0 +1,48 @@
+  .opacity_1{
+    opacity: $opacity_1;
+  }
+  .opacity_2{
```
```suggestion
  .opacity_2 {
```
You get the picture 😉

jasoncypret commented on playbook-website/app/javascript/site_styles/_visual_guidelines.scss:
```diff
@@ -41,4 +42,49 @@
+  align-items: center;
+  margin: none;
+  background: none;
+  border-radius: none;
+  border: 0px;
+  padding: none;
+  flex-wrap: wrap;
+  }
+}
+  .opacity_1{
+    opacity: .1;
+  }
+  .opacity_2{
+    opacity: .2;
+  }
+  .opacity_3{
+    opacity: .3;
+  }
+  .opacity_4{
+    opacity: .4;
+  }
+  .opacity_5{
+    opacity: .5;
+  }
+  .opacity_6{
+    opacity: .6;
+  }
+  .opacity_7{
+    opacity: .7;
+  }
+  .opacity_8{
+    opacity: .8;
+  }
+  .opacity_9{
+    opacity: .9;
+  }
+  .opacity_10{
+    opacity: 1;
+  }
+}
```
```suggestion
```

Why is this included twice?

jasoncypret commented on playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb:
```diff
@@ -0,0 +1,77 @@
+}) %><br>
+
+<%= pb_rails("card", props: {shadow: "deep", margin_bottom: "xl", padding: "none", padding_top: "md" }) do %>
+<ul class="opacity-play pb--utlities-color">
+<li data-name="Data 1" data-variable="$data_1">
+<div class="pb--color_data_1 opacity_1"></div>
+<p class="name">Opacity_1</p>
+<p class="var">$opacity_1</p>
+</li>
+<li data-name="Data 1" data-variable="$data_1">
+<div class="pb--color_data_1 opacity_2">
+</div>
+<p class="name">Opacity_2</p>
+<p class="var">$opacity_2</p>
+</li>
+<li data-name="Data 1" data-variable="$data_1">
+<div class="pb--color_data_1 opacity_3">
+</div>
+<p class="name">Opacity_3</p>
+<p class="var">$opacity_3</p>
+</li>
+<li data-name="Data 1" data-variable="$data_1">
+<div class="pb--color_data_1 opacity_4">
+</div><p class="name">Opacity_4</p>
+<p class="var">$opacity_4</p>
```
You have 0 indentations here. I can't tell what is a container. Please correct this whole file.

Tab to indent
Shift + Tab to outdent

thestephenmarshall commented on playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:
```diff
@@ -0,0 +1,48 @@
+ul {
+   
+  &.opacity-play {
```
Not sure I like this selector name. What are we styling here? Can we be more specific? `-play` seems too generic and doesn't tell me a lot about the context.

cvargaspower replied:
We aren't styling anything, the way I built the opacity circles, is I used "pb--utlities-color", but it was giving it margins and borders so I used "opacity-play" to get rid of all of that. I am not sure what to name it ;u; '' Do you have a suggestion?

thestephenmarshall commented on playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb:
```diff
@@ -0,0 +1,74 @@
+   <pre class="highlight">Example in SCSS<p><code>@import "tokens/opacity";</code></p></pre>
+</div>
+<% end %>  
+<% end %>
```
```suggestion
<% end %>

```

The ⛔️ at the end of the file here
<img width="144" alt="Screen Shot 2021-11-19 at 8 50 39 AM" src="https://user-images.githubusercontent.com/2293844/142642318-053c06f0-377c-43a9-b673-f7435ca79d01.png">

indicates a new line *must* be added.

thestephenmarshall reviewed (dismissed):
Needs work still. Let's keep on this PR.

thestephenmarshall commented on playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:
```diff
@@ -0,0 +1,55 @@
+ul.opacity-removing-utilities {
+
+  &.opacity-removing-utilities {
```
@cvargaspower this would translate to:

```html
<ul class="opacity-removing-utilities opacity-removing-utilities">
```

I don't know that this was your intent?

thestephenmarshall commented on playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:
```diff
@@ -0,0 +1,55 @@
+  .opacity_10 {
+    opacity: $opacity_10;
+  }
+  
```
```suggestion
  
  
```

thestephenmarshall replied:
Not sure if this is needed now

thestephenmarshall commented on playbook-website/app/javascript/site_styles/_visual_guidelines.scss:
```diff
@@ -1,25 +1,30 @@
+
   .positioning-guideline {
-    & > * {
+    &>* {
```
🤔

cvargaspower replied:
That isn't me? I think Dat is doing positioning.

cvargaspower replied:
I am not sure what I am supposed to do here. This was here before I made changes to the file.

thestephenmarshall commented on playbook-website/app/javascript/site_styles/_visual_guidelines.scss:
```diff
@@ -31,14 +36,96 @@
+    }
+  }
+
+  ul.opacity-removing-utilities {
+
+    &.opacity-removing-utilities {
+      list-style: none;
+      display: flex;
```
Doesn't this exist in the other file above?

cvargaspower replied:
Yes, as I said in my notes: I made an opacity_guidelines.scss but it would not connect to the opacity.html.erb and only worked when I put it underneath the visual_guidelines.scss for some reason. Paul tried to help me but we both were stumped. ... maybe someone else can help.

thestephenmarshall reviewed (commented):
Getting there

jasoncypret commented on playbook-website/app/javascript/site_styles/_visual_guidelines.scss:
```diff
@@ -1,6 +1,7 @@
 @import "playbook-ui/dist/tokens/spacing";
 @import "playbook-ui/dist/tokens/screen_sizes";
 @import "playbook-ui/dist/tokens/opacity";
+@import "./opacity_guidelines"
```
@import "opacity_guidelines"

jasoncypret commented on playbook-website/app/javascript/site_styles/_visual_guidelines.scss:
```diff
@@ -35,15 +41,41 @@
-    .eight { margin-left: 350px; }
-    .nine { margin-left: 400px; }
-    .ten { margin-left: 450px; }
+    .two {
+      margin-left: 50px;
+    }
+
+    .three {
+      margin-left: 100px;
+    }
+
+    .four {
+      margin-left: 150px;
+    }
+
+    .five {
+      margin-left: 200px;
+    }
+
+    .six {
+      margin-left: 250px;
+    }
+
+    .seven {
+      margin-left: 300px;
+    }
+
+    .eight {
+      margin-left: 350px;
+    }
+
+    .nine {
+      margin-left: 400px;
+    }
+
+    .ten {
+      margin-left: 450px;
+    }
```
```suggestion
```

Unless you are utilizing this it has been removed from master

jasperfurniss replied:
@cvargaspower This all needs to match what is on Master. You just need to commit Jason's suggestion. I think you only need really the one line of code in this file on line 3: `@import "playbook-ui/dist/tokens/opacity";`

jasoncypret commented on playbook-website/app/javascript/site_styles/_visual_guidelines.scss:
```diff
@@ -10,18 +12,22 @@
     margin: auto;
   }
-  @include break_at(breakpoint("sm")){
+
```
```suggestion
```

jasoncypret commented on playbook-website/app/views/pages/visual_guidelines.html.erb:
```diff
@@ -14,6 +14,7 @@
 
   <%= pb_rails("background", props: { classname: "token-wrapper", background_color: "light", padding: "xl" }) do %>
       <%= render "pages/visual_guidelines_partials/colors.html.slim" %>
+      <%= render "pages/visual_guidelines_partials/opacity.html.erb" %>
```
```suggestion
      <%= render "pages/visual_guidelines_partials/opacity" %>
```

Extension is not needed

jasoncypret commented on playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb:
```diff
@@ -0,0 +1,75 @@
+      <p class="var">$opacity_10</p>
+   </li>
+</ul>
+<% react_contents = Rails.root.join("app/views/pages/code_snippets/user_jsx.txt").read %>
+<%= pb_rails("background", props: { background_color: "dark", padding: "xs", id: "code-wrapper", padding_left: "lg" }) do %>
+<div class="pb--codeCopy" data-action="toggle" data-togglable="code_example">
+   <pre class="highlight">Example in SCSS<p><code>@import "tokens/opacity";</code></p></pre>
+</div>
```
Check out how the other files are doing this code sample and replicate. Move the react contents variable to the pages controller as well. Use an actual code sample with an opacity token in action. See the others for example

jasoncypret reviewed (changes requested):
You need to get the latest from master. I cleaned up all the files and now you have better examples of how this should be.

jasoncypret commented on playbook-website/app/javascript/site_styles/_visual_guidelines.scss:
```diff
@@ -1,8 +1,17 @@
 @import "playbook-ui/dist/tokens/spacing";
 @import "playbook-ui/dist/tokens/screen_sizes";
+@import "playbook-ui/dist/tokens/opacity";
```
Still need this file cleaned up

jasoncypret commented on playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb:
```diff
@@ -0,0 +1,76 @@
+<h1 aria-label="online" class="pb_title_kit_1">Opacity</h1>
+<br>
```
Lets get rid of break tags, use spacing props and title kits

jasoncypret commented on playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb:
```diff
@@ -0,0 +1,76 @@
+      <p class="var">$opacity_10</p>
+   </li>
+</ul>
+<% react_contents = Rails.root.join("app/views/pages/code_snippets/opacity_tokens.txt").read %>
+    
+<%= pb_rails("background", props: { background_color: "dark", padding: "xs", id: "code-wrapper" }) do %>
+  <%= pb_rails("caption", props: { padding_x: "md", padding_top: "md", text: "Example in Sass" }) %>
+    <div class="pb--codeCopy" data-action="toggle" data-togglable="code_example">
+      <pre class="highlight"><%= raw rouge(react_contents, "react") %></pre>
+    </div>    
+<% end %>  
```
Let's improve this section. Take a look at this: https://github.com/powerhome/playbook/blob/master/playbook-website/app/views/pages/visual_guidelines_partials/_positioning.html.erb#L31-L36

1. Move the `Rails.root.join("app/views/pages/code_snippets/opacity_tokens.txt").read` to the pages controller
2. Match the other styling/settings referenced in the file above.

jasoncypret reviewed (commented):
I don't think my requested changes were addressed. 
1. [Your SASS file add a bunch of unused styles](https://github.com/powerhome/playbook/blob/d381b470a3e97ef3d8cbce6c55f348154495ee1c/playbook-website/app/javascript/site_styles/_visual_guidelines.scss) see [this file](https://github.com/powerhome/playbook/blob/master/playbook-website/app/views/pages/visual_guidelines_partials/_positioning.html.erb) for example. 
2. [Your erb file](https://github.com/powerhome/playbook/blob/d381b470a3e97ef3d8cbce6c55f348154495ee1c/playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb) doesn't match the patterns from the other files. It contains break tags instead of using spacing props and don't use kits for the title. [Here's an example](https://github.com/powerhome/playbook/blob/master/playbook-website/app/views/pages/visual_guidelines_partials/_positioning.html.erb) of how the top should look.

jasperfurniss commented on playbook/dist/playbook.css:
```diff
@@ -34,6 +34,8 @@ Background colors ------------------*/
     background: #F3F7FB; }
   [class^=pb_background_kit].pb_background_color_white {
     background: #FFFFFF; }
+<<<<<<< HEAD
```
You gotta get rid of these first.

jasperfurniss commented on playbook/dist/playbook.css:
```diff
@@ -102,6 +104,7 @@ Background colors ------------------*/
     background: #9E64E9; }
   [class^=pb_background_kit].pb_background_color_insulation {
     background: #FF2229; }
+>>>>>>> 0de8ca96b833aca94a18f93ba968fad69d09a3b5
```
And get rid of this.

jasperfurniss commented on playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb:
```diff
@@ -0,0 +1,70 @@
+    <div class="pb--codeCopy" data-action="toggle" data-togglable="code_example">
+      <pre class="highlight"><%= raw rouge(@opacity_code_example, "react") %></pre>
+    </div>    
+<% end %>
```
Can you indent lines 12-63. And then also un-indent lines 66-68 by one tab.

jasoncypret commented on playbook-website/app/javascript/site_styles/_visual_guidelines.scss:
```diff
@@ -19,13 +28,69 @@
+      }
+    }
+
+    .two {
+      margin-left: 50px;
+    }
+
+    .three {
+      margin-left: 100px;
+    }
+
+    .four {
+      margin-left: 150px;
+    }
+
+    .five {
+      margin-left: 200px;
+    }
+
+    .six {
+      margin-left: 250px;
+    }
+
+    .seven {
+      margin-left: 300px;
+    }
+
+    .eight {
+      margin-left: 350px;
+    }
+
+    .nine {
+      margin-left: 400px;
+    }
+
+    .ten {
+      margin-left: 450px;
+    }
+  }
+
```
```suggestion
```

jasoncypret reviewed (changes requested):
I marked what needs to be done to get this PR in. You should just have to accept and make sure you don't have any merge conflicts

thestephenmarshall commented:
@nickamantia this one has been very long in the tooth and @jasoncypret needs to re-review here in order to get it moving.

jasoncypret commented:
@nickamantia I'm having trouble viewing the Milano deploy.

<img width="1790" alt="image" src="https://user-images.githubusercontent.com/863031/153948569-a314336b-fd11-4300-bf47-000b270471a1.png">

jasoncypret reviewed (changes requested):
I don't think this is related to yours but this section is now missing it's background. 
<img width="1233" alt="image" src="https://user-images.githubusercontent.com/863031/154110077-c5c6bb03-849c-469d-9833-469fac89ba52.png">

Can you fix it up with this PR?

nickamantia commented:
Line Height background fix:

![Screen Shot 2022-02-16 at 11 55 34 AM](https://user-images.githubusercontent.com/92755007/154316203-4736f211-7a49-4c09-bdbb-22683c509b56.png)
